_plugins/fetch_docs.rb

```diff
@@ -176,8 +176,8 @@ def gen_versions_xml_file(filePath, sourceData)
     %x( mkdir -p docs/infinispan-operator/#{version}/ )
     %x( mkdir -p docs/infinispan-operator/topics/images/ )
     %x( mv _optmp/*.html _optmp/**/documentation/asciidoc/css _optmp/**/documentation/asciidoc/js "docs/infinispan-operator/#{version}/" )
-    #Use images only on master. Causes harmless "cannot stat" messages.
-    %x( cp -r _optmp/infinispan-operator-master/documentation/asciidoc/topics/images/* "docs/infinispan-operator/topics/images/" )
+    #Use images only on main. Causes harmless "cannot stat" messages.
+    %x( cp -r _optmp/infinispan-operator-main/documentation/asciidoc/topics/images/* "docs/infinispan-operator/topics/images/" )
```

Comment:
operator main or 2.1.x ?

Reply:
this is for the images only, which we get from the main branch and use for all versions. I probably need to look into this and improve it but it works for now. there's only one image btw.